packages/pangea-sdk/pangea_django/README.md

```diff
@@ -0,0 +1,34 @@
+# Pangea Django Setup
+
+Use Django as your authentication backend in a few simple steps.
+
+## Install the Pangea SDK
+
+Run the following command
+
+```
+pip install pangea-sdk
+```
+
+## Set up AutnN with Pangea
+
+Go to the [Pangea AuthN service](https://console.pangea.cloud/service/authn) and step through the dialogs. You will need to collect the auth token you created, the domain and the hosted login link for later. Also set the redirect (callback) link under Settings/General/Redirect(callback) Settings, this is where the hosted flow will send the user once they are logged in.
+
+## Environment Variabled
+Set up the environment variables ([Instructions](https://pangea.cloud/docs/getting-started/integrate/#set-environment-variables)) `PANGEA_AUTHN_TOKEN` and `PANGEA_DOMAIN` with your project token configured on the Pangea User Console (token should have access to AuthN service [Instructions](https://pangea.cloud/docs/getting-started/configure-services/#configure-a-pangea-service)) and with your Pangea domain.
+
+## Django Settings
+
+Modify the following values in the settings.py of your Django project
+
+* Add `'pangea_django.PangeaAuthMiddleware'` to the `MIDDLEWARE`. You can keep or remove the standard Django contrib auth middleware.
+* Ensure you have sessions set up, you should have this already. See [Using Django Sessions](https://docs.djangoproject.com/en/5.0/topics/http/sessions/)
```

Comment:
last lts release? 4.2?

Reply:
The link is good and its what comes up when i google for django session